tests/Instances/Data/VectorNonEmpty.hs

```diff
@@ -0,0 +1,26 @@
+{-# LANGUAGE DeriveGeneric      #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+{-|
+Module:      Instances.Data.Vector
+Copyright:   (C) 2014-2017 Ryan Scott
+License:     BSD-style (see the file LICENSE)
+Maintainer:  Ryan Scott
+Stability:   Provisional
+Portability: GHC
+
+Provides 'Arbitrary' instances for 'Vector' types.
+-}
+module Instances.Data.VectorNonEmpty () where
```

Comment:
For consistency with how `nonempty-vector` names its modules, let's name this module `Instances.Data.Vector.NonEmpty` instead.